saleor/graphql/checkout/mutations.py

```diff
@@ -287,6 +287,7 @@ def save(cls, info, instance: models.Checkout, cleaned_input):
                         "Can't create checkout with unpublished product.",
                         code=exc.code,
                     )
+            info.context.plugins.checkout_quantity_changed(instance)
```

Comment:
Let's discuss if this is needed here - we create a new checkout instance, so should we call a "quantity changed" hook? If anything, I would expect "checkout created" hook, if we had it.

Reply:
It will be removed in separate PR.